src/app/auth/login/login.service.ts

```diff
@@ -125,6 +125,7 @@ export class LoginService {
     }
 
     private redirectUrlLoginCallback(loginResponse: LoginResponse): void {
+        this.localStorageService.setAccountId(loginResponse.account.id);
```

Comment:
We already have `account$` event, and `LoggedUserService` is a dedicated service for managing user data. Can't we encapsulate it there?